app/views/settings/mastodon_authentication.html.erb

```diff
@@ -0,0 +1,21 @@
+<% content_for :subnav do %>
+  <a href="/settings">Back to Settings</a>
+<% end %>
+
+<div class="box wide">
+  <%= form_with url: "/settings/mastodon_auth", method: :get do |f| %>
+    <p>
+    Enter the name of your Mastodon instance without the scheme, 
+    i.e mastodon.social not https://mastodon.social
+    </p>
+
+    <div class="boxline">
+      <%= f.label :mastodon_instance_name, "Mastodon Instance:", :class => "required" %>
```

Comment:
We have a few `placeholder` attributes in the code, I think this would be a good place for one. With the way you wrote `sanitized_instance_name`, you don’t even need to include the instruction above about not including `https`.

Reply:
Ah yes, good point! I added `sanitized_instance_name` later. I'll await further review before making any changes, but I've noted this.